nemo/collections/nlp/models/text_classification/text_classification_model.py

```diff
@@ -62,11 +62,18 @@ def __init__(self, cfg: DictConfig, trainer: Trainer = None):
             config_file=self.register_artifact('language_model.config_file', cfg.language_model.config_file),
             config_dict=cfg.language_model.config,
             checkpoint_file=cfg.language_model.lm_checkpoint,
+            nemo_file=self.register_artifact('language_model.nemo_file', cfg.language_model.get('nemo_file', None)),
```

Comment:
It seems strange to me that we're passing a .nemo file as an arg here. Why are we doing this?

Reply:
This also has the effect that the nemo file for the text classification model will have a nemo file inside of it.

Reply:
I tested saving the fine tune the model to nemo file and load it to run inference. It works fine.

```python
model_ner.save_to('test.nemo')
tt = nemo_nlp.models.TokenClassificationModel.restore_from('./test.nemo', trainer=trainer)
tt.half().evaluate_from_file(
    text_file=os.path.join(NER_DATA_DIR, 'sample_text_dev.txt'),
    labels_file=os.path.join(NER_DATA_DIR, 'sample_labels_dev.txt'),
    output_dir=exp_dir,
    add_confusion_matrix=False,
    normalize_confusion_matrix=True,
    batch_size=1
)
```